src/Compute.js

```diff
@@ -0,0 +1,215 @@
+import React, { useState } from 'react'
+
+import { asset } from './asset'
+import { algoAsset, createComputeService, rawAlgoMeta } from './compute-asset'
+
+export default function Compute({ ocean, web3 }) {
+  const [ddoAssetId, setDdoAssetId] = useState('')
+  const [jobStatus, setJobStatus] = useState('')
+  const [jobId, setJobId] = useState('')
+  const [agreementId, setAgreementId] = useState('')
+  const [ddoAlgorithmId, setDdoAlgorithmId] = useState('')
+  const [divAlgoStyle, setdivAlgoStyle] = useState('')
```

Comment:
This could probably be a boolean (like `isAlgoInputVisible` or something) so you don't have to store the raw style. Then you can set the correct style based on that boolean in returned JSX

Reply:
Yeah, but still need to keep 'none' or 'block' to divAlgoStyle.
Feel free to make a suggestion though.

Thanks

Reply:
Like I said, we can keep boolean here and when applying style do something like `style={{display: isAlgoInputVisible ? 'block' : 'none'}}`

Reply:
got it